### PR TITLE
App updated to newest android version

### PR DESCRIPTION
**Pull request by DanGavrielov**
application now support Android API 34
migrated from gcm to fcm

### CODE REVIEW

File: app/keystore.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+storePassword=123456
```
Comment (yuvalp-k2view) [inline app/keystore.properties]: we shouldb't have keys in the repository --- it's public!

File: app/src/main/java/ruon/android/util/UserLog.java [hunk 2]
```diff
@@ -1,9 +1,10 @@
 package ruon.android.util;
 
+import android.os.Build;
 import android.util.Log;
 
 public class UserLog {
-	public static final boolean DEBUG = false;
+	public static final boolean DEBUG = true;
```
Comment (yuvalp-k2view) [inline app/src/main/java/ruon/android/util/UserLog.java]: is this okay?

**Review by yuvalp-k2view — CHANGES_REQUESTED** [review]

File: app/keystore.properties (hunk 1 above)
Comment (DanGavrielov) [reply app/keystore.properties]: key was removed in last commit, it didn't work anyway as I need the original signing key to package an APK for GooglePlay

File: app/src/main/java/ruon/android/util/UserLog.java (hunk 2 above)
Comment (DanGavrielov) [reply app/src/main/java/ruon/android/util/UserLog.java]: see last commit

**Review by yuvalp-k2view — APPROVED** [review]

**Pull request merged**